lib/ansible/modules/cloud/digital_ocean/digital_ocean_floating_ip_facts.py

```diff
@@ -98,41 +86,34 @@ def core(module):
     page = 1
     has_next = True
     floating_ips = []
-    while has_next or 200 != status_code:
+    status_code = 400
```

Comment:
if you want to initialise wouldn't be better to set it to None?

Reply:
ok.